CHANGELOG.md

```diff
@@ -3,16 +3,17 @@
 **This project is in Release Candidate stage.**
 
 ### Enhancements
-* None
+* Improve performance of interprocess mutexes on iOS which don’t need to support reader-writer locking. The primary beneficiary of this is beginning and ending read transactions, which is now almost as fast as pre-v13.0.0 (Core upgrade).
 
 ### Fixed
 * None
 
 ### Compatibility
 * Realm Studio: 13.0.0 or later.
+* Fileformat: Generates files with format v23. Reads and automatically upgrade from fileformat v5 (Core upgrade).
```

Comment:
I think we should remove this line? We mentioned it on the Core 13 upgrade changelog,  but now we are just bumping a minor version of Core.